requirements.txt

```diff
@@ -51,30 +64,12 @@ beautifulsoup4>=4.8.0
 
 django-simple-cookie-consent>=0.1.1
 
-
 importlib-resources>=1.0.2
 
 html5lib>=1.0.1
 Pillow>=3.0
+backport_collections==0.1
+lxml
 six
 pytz
-
-#aldryn-categories
-django-parler
-aldryn-translation-tools
-
-#aldryn-newsblog
-python-dateutil
```

Comment:
This is required for newsblog so it cant be removed

Reply:
You have 43 unapplied migration(s). Your project may not work properly until you apply the migrations for app(s): aldryn_categories, aldryn_newsblog, aldryn_people, bootstrap4_alerts, bootstrap4_badge, bootstrap4_card, bootstrap4_carousel, bootstrap4_collapse, bootstrap4_content, bootstrap4_grid, bootstrap4_jumbotron, bootstrap4_link, bootstrap4_listgroup, bootstrap4_media, bootstrap4_picture, bootstrap4_tabs, bootstrap4_utilities, djangocms_googlemap, djangocms_icon.
Run 'python manage.py migrate' to apply them.